docs/eng/contributing.md

```diff
@@ -0,0 +1,43 @@
+# Contributing
+
+## Project structure
+
+1. Each module should have the following structure:
+    ```
+    module-name/
+        CmakeLists.txt
+        src/
+            file.h
+            ...
+        include/
+            file.c
```

Comment:
shouldn't `.h` files be in `include` folder and `.c` in `src`?

Reply:
fixed in [7adb5f8](https://github.com/ThatDraenGuy/motorland/pull/8/commits/7adb5f895c54bff638100a9fa85536c008456044)